src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -485,7 +542,11 @@ class DocAnnotator extends Annotator {
             this.highlightThrottleHandle = null;
         }
 
-        if (this.hasTouch && this.isMobile) {
+        if (!this.canAnnotate) {
```

Comment:
This is now stored as `this.permissions.canAnnotate`. Update this around the code! See https://github.com/box/box-content-preview/blob/master/src/lib/annotations/Annotator.js#L70